### ISSUE TITLE
JsonValue is not owned

### ISSUE THREAD

**Issue by vitiral** [anguslees/rust-jsonnet#7]
Hey, thanks for the great library.

`JsonValue` is not currently owned, as it requires a lifetime. The docs say it is owned though :smile: 

https://docs.rs/jsonnet-rs/0.6.0/jsonnet/struct.JsonValue.html

**Comment by anguslees** [comment]
Yes, true - in the sense that it's not `'static`.  I was trying to communicate that `JsonValue` frees the underlying jsonnet value when the `JsonValue` object is dropped - unlike `JsonVal` which just "borrows" the underlying jsonnet value.  The `JsonValue` lifetime points to the parent `JsonnetVm` and not the internally-referenced jsonnet value.

I agree that's confusing when a reader (quite reasonably!) assumes a strict interpretation of "owned" and I should clarify that further in the docs.
